db/settings.go

```diff
@@ -5,10 +5,22 @@ type Settings struct {
 	Token     string
 }
 
+type Getchatid struct {
+	chatId int64
```

Comment:
This file seems to be the wrong place for adding dotsave db helper functions. Please create a separate file for misc module.

Reply:
Ok